src/util/mapFolderData.js

```diff
@@ -3,25 +3,50 @@ import { getElapsedTime } from "./getElapsedTime";
 
 export const mapFolderData = (folder) => {
   if (!folder) return [];
-  const { name, owner, links } = folder;
+  const { name, owner, data, links, folder_id } = folder;
 
-  const mapLinks = (link) => {
-    const { id, createdAt, url, imageSource, title, description } = link;
-    return {
-      id,
-      url,
-      imageSource,
-      alt: `${title ?? url}의 대표 이미지`,
-      elapsedTime: getElapsedTime(createdAt),
-      description,
-      createdAt: format(new Date(createdAt), "yyyy. MM. dd"),
-    };
+  const mapLinks = (data) => {
+    if (!links) {
+      const {
+        id,
+        name,
+        created_at,
+        url,
+        image_source,
+        folder_id,
+        title,
+        description,
+      } = data;
+      return {
+        id,
+        url,
+        name,
+        imageSource: image_source,
+        folder_id,
+        alt: `${title ?? url}의 대표 이미지`,
+        elapsedTime: getElapsedTime(created_at),
+        description,
+        createdAt: format(new Date(created_at), "yyyy. MM. dd"),
+      };
+    } else {
+      const { id, createdAt, url, imageSource, title, description } = data;
+      return {
+        id,
+        url,
+        imageSource,
+        alt: `${title ?? url}의 대표 이미지`,
+        elapsedTime: getElapsedTime(createdAt),
+        description,
+        createdAt: format(new Date(), "yyyy. MM. dd"),
+      };
+    }
   };
-
   return {
     profileImage: owner?.profileImageSource,
     ownerName: owner?.name,
     pageName: name,
-    links: links?.map(mapLinks) ?? [],
+    folderId: folder_id,
+    links: (data || links)?.map(mapLinks) ?? [],
+    tagData_links: (data || links)?.map(mapLinks) ?? [],
```

Comment:
```suggestion
    links: validLinks?.map(mapLinks) ?? [],
    tagData_links: validLinks?.map(mapLinks) ?? [],
```

이런식으로 하나의 변수에 담아두는게 관리 포인트를 줄일 수 있는 방법입니다.